src/Playwright/build/Microsoft.Playwright.targets

```
@@ -3,14 +3,18 @@
   <Target Name="CopyPlaywrightFilesToOutput" AfterTargets="BeforeBuild">
     <PropertyGroup>
       <PlaywrightPlatform Condition="'$(PlaywrightPlatform)' == ''">%(RuntimePack.RuntimeIdentifier)</PlaywrightPlatform>
-      <PlaywrightPlatform Condition="'$(PlaywrightPlatform)' == '' AND $([MSBuild]::IsOSPlatform('Linux'))">linux</PlaywrightPlatform>
+      <PlaywrightPlatform Condition="'$(PlaywrightPlatform)' == '' AND $([MSBuild]::IsOSPlatform('Linux')) AND '$([System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture)' == 'X64'">linux</PlaywrightPlatform>
```

Maybe use this instead?
```suggestion
      <PlaywrightPlatform Condition="'$(PlaywrightPlatform)' == '' AND $([MSBuild]::IsOSPlatform('Linux')) AND '$([System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture)' != 'Arm64'">linux</PlaywrightPlatform>
```

---

Not sure, better to have no driver at all rather than having one for a different architecture on e.g. arm (e.g. normal arm). wdyt?

---

The current behavior is that the linux one will be chosen regardless of the architecture so it is "backwardcompatible" 😄

---

but having a not functioning driver since you can't run a driver for a different arch is also bad. 😄